variables.tf

```diff
@@ -0,0 +1,34 @@
+variable "domain_name" {
+  type = string
+
+  description = <<EOS
+Domain name which shall be used as sender domain by SES.
+EOS
+}
+
+variable "route53_zone_id" {
```

Comment:
in newer modules, we avoid using strings when passing _existing_ object, but pass the whole object: `object({ id = string })`